### PR TITLE
feat(backend): 25 implement reviews

### PR DESCRIPTION
**Pull request by DeceaCezara**

### CODE REVIEW

File: backend/src/main/java/com/backend/Model/Review.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    private Integer rating;
+
+    private String description;
+
+    @Override
+    public String toString() {
+        return "Review{" +
+                "id=" + reviewId +
+                ", book=" + book.getTitle() +  // afișăm titlul cărții pentru simplitate
```
Comment (GotaSeptimiuAndrei) [inline backend/src/main/java/com/backend/Model/Review.java]: sa lasam commenturi in engleza

File: backend/src/main/java/com/backend/Model/Review.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+public class Review {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "review_id")
+    private Long reviewId;
+
+    @ManyToOne
+    @JoinColumn(name = "book_id", nullable = false)
+    private Book book;
```
Comment (GotaSeptimiuAndrei) [inline backend/src/main/java/com/backend/Model/Review.java]: book_id e long, dc ai pus Book

File: backend/src/main/java/com/backend/Model/Review.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+    @ManyToOne
+    @JoinColumn(name = "user_id", nullable = false)
+    private User user;
```
Comment (GotaSeptimiuAndrei) [inline backend/src/main/java/com/backend/Model/Review.java]: same thing

**Review by B0gsto — APPROVED** [review]

**Review by GotaSeptimiuAndrei — APPROVED** [review]
ok

**Pull request merged**